tests/unittests/test_handler/test_schema.py

```diff
@@ -447,4 +447,23 @@ def test_all_integration_test_cloud_config_schema(self):
         if errors:
             raise AssertionError(', '.join(errors))
 
+
+def _get_schema_doc_examples():
+    examples_dir = Path(
+        cloudinit.__file__).parent.parent / 'doc' / 'examples'
+    assert examples_dir.is_dir()
+
+    all_text_files = (f for f in examples_dir.glob('cloud-config*.txt')
+                      if not f.name.startswith('cloud-config-archive'))
+    return all_text_files
+
+
+class TestSchemaDocExamples:
+    schema = get_schema()
```

Comment:
This is a nit (so feel free to not address it), but `TestCloudConfigExamples` has `SCHEMA` in all-caps and referred to as `TestCloudConfigExamples.SCHEMA` in its test: it would be nice to be consistent.

Reply:
Would it be ok if I updated the other class? Class variables usually don't use all caps unless they're constants, and `self` is a less verbose way of accessing a class variable than typing out the whole class every time.

Reply:
> Would it be ok if I updated the other class?

In general, I expect us all to have ownership of all parts of the codebase, so the answer is yes. In this specific case, the code in question was added by @lucasmoura this week, so lets give him an opportunity to weigh in on what his thinking was.

> Class variables usually don't use all caps unless they're constants

This is a constant, in the sense that we only expect to read from it, but I don't feel strongly about this either way.

> `self` is a less verbose way of accessing a class variable than typing out the whole class every time.

Yep, I'd also be fine with this. I can see using the class name makes sense if we're really treating classes purely as namespaces (in which use `self` shouldn't have any special meaning), but I agree that it is an unusual spelling (and also makes renaming this class more work, for example).

Reply:
Sorry for the long delay, but I don't have any problems with the proposed change. Originally I thought about the schema as a constant, but I do agree that using self for that particular use case is less verbose